cmd/soroban-cli/src/commands/contract/invoke.rs

```diff
@@ -293,6 +291,23 @@ impl Cmd {
             })
             .transpose()
     }
+
+    fn send(&self, sim_res: &SimulateTransactionResponse) -> Result<ShouldSend, Error> {
+        Ok(match self.send {
+            Send::Default => {
+                if self.is_view {
+                    ShouldSend::No
```

Comment:
`--is-view` is basically ignored if a non-default send mode is set, since `--send` is the new option it should take precedence. This code got reorganized so that the combination of the two options can set the `ShouldSend` enum value which was easier to do when the mapping isn't happening in the `Send` enum's fn.